addon/-private/system/store.js

```diff
@@ -2432,6 +2432,16 @@ function _commit(adapter, store, operation, snapshot) {
       if (adapterPayload) {
         payload = normalizeResponseHelper(serializer, store, typeClass, adapterPayload, snapshot.id, operation);
         if (payload.included) {
+          // first find possibly uncommitted records from this inclusion
+          var types = payload.included.map(function (record) { return record.type; });
+          types.forEach(function (type) {
+            var existingRecords = store.peekAll(type);
+            var uncommitted = existingRecords.filter(function (record) {
+              return record.get('isNew') && !record.get('isSaving');
```

Comment:
correct. It does. There currently is no test that fails showcasing that scenario, but it's easy to write one. 

Problem is, I am stumbling with a way to let the store know which records are being persisted to the backend. Should the EmbeddedRecordMixin flag the records by changing their currentState to 'inFlight'? How should I handle this in the Ember data runloop?